### DISCUSSION TITLE
Using Selenium/ChromeDriver to test WPF application with multiple embedded browsers

### DISCUSSION THREAD

**Discussion by rperneel** [cefsharp/CefSharp#4995] [General]
Hello,

I have an application that has multiple Cef browsers embedded into it, but I am unable to connect to the app with the chromedriver that matches the Cef version.  

If I only have one browser active, I am able to connect to the debugging port that is set in my CefSettings.

Is there a way to support this?

Thanks.

**Comment by amaitland** [comment]
Do multiple pages show up in `chrome://inspect` if you use `Chrome`?

Unfortunately that sounds like a bug in `CEF`. You can search the issue tracker at https://github.com/chromiumembedded/cef/issues

**Comment by rperneel** [comment]
Thanks for the info.  will look there for insight.
